modules/templates.js

```diff
@@ -0,0 +1,13 @@
+angular.module('pages').run(['$templateCache', function($templateCache) {
```

Comment:
This filename should be in the same place as the last one, no? also this module is `pages` but it should be `feincms-pages.templates`. The diff here should be a change to `modules/pages/scripts/templates.js` that makes it look like this file